src/styles/index.css

```diff
@@ -0,0 +1,238 @@
+.cards-container {
+  box-sizing: border-box;
+  --column-count: 1;
+  padding: 40px 50px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 48px 46px;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 488px) {
+  .cards-container {
+    --column-count: 2;
+  }
+}
+@media (min-width: 768px) {
+  .cards-container {
+    --column-count: 3;
+  }
+}
+@media (min-width: 1024px) {
+  .cards-container {
+    --column-count: 4;
+  }
+}
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 16px;
+  height: auto;
+  transition: transform 300ms, color 300ms;
+}
+.card__image {
+  display: flex;
+}
+.card__image .image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+  background-image: url(../images/imac.jpeg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+.card__title {
+  margin: 40px 16px 0;
+  line-height: 18px;
+  height: 36px;
+  width: 166px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+}
+.card__product-code {
+  margin-top: 4px;
+  padding: 0 16px;
+  width: 100%;
+  justify-content: center;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+.card__reviews {
+  display: flex;
+  margin: 16px;
+  margin-bottom: 0;
+  width: 166px;
+  justify-content: space-between;
+}
+.card__reviews .reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 16px;
+  color: #060b35;
+  text-align: right;
+}
+.card__price {
+  display: flex;
+  width: 166px;
+  line-height: 18px;
+  justify-content: space-between;
+  margin: 24px 16px 0;
+}
+.card__price .price {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+}
+.card__price .amount {
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+}
+.card .button {
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  width: 166px;
+  height: 40px;
+  margin: 16px;
+  border-color: #00acdc;
+  border-radius: 5px;
+  background: #00acdc;
+  transition: background-color 300ms, border 300ms;
+}
+.card .button__buy {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: white;
+  font-size: 14px;
+  font-weight: 700;
+  transition: color 300ms;
+}
+.card .button__buy:hover {
+  color: #00acdc;
+}
+.card .button:hover {
+  background-color: white;
+  border: 1px solid #00acdc;
+}
+.card:hover {
+  transform: scale(1.2);
+}
+.card:hover .card__title {
+  color: #34568b;
+}
+
+.header {
+  width: 100vw;
```

Comment:
Using `100vw` for width can cause horizontal scrolling issues due to the scrollbar width. Consider using `100%` instead to avoid this potential issue.